docs/Wallets/WalletsOverview.md

```diff
@@ -4,25 +4,45 @@ The Loki wallet is a gateway to private decentralised transactions and communica
 
 >> TO DO: What do Wallets do?
 
-#### Web Wallet (Loki Locker)
-The quickest way to access a Loki Wallet is through a web based wallet, [Loki Locker](https://lokilocker.com/). The [Web Wallet](../Wallets/WebWallet/WebWalletOverview.md) is the easiest to use and doesn't require any downloaded files.
+## Web Wallet (Loki Locker)
+The quickest way to access a Loki Wallet is through a web based wallet, [Loki Locker](https://lokilocker.com/). The Web Wallet is the easiest to use and doesn't require any downloaded files.
```

Comment:
this should not be recommended until it works

Reply:
I was unaware that the loki locker wasnt working

Reply:
Removed content. Added to do.